src/linkplay/bridge.py

```diff
@@ -83,10 +83,19 @@ def __init__(self, bridge: LinkPlayBridge):
 
     async def update_status(self) -> None:
         """Update the player status."""
-        self.properties = await self.bridge.json_request(LinkPlayCommand.PLAYER_STATUS)  # type: ignore[assignment]
-        self.properties[PlayerAttribute.TITLE] = decode_hexstr(self.title)
-        self.properties[PlayerAttribute.ARTIST] = decode_hexstr(self.artist)
-        self.properties[PlayerAttribute.ALBUM] = decode_hexstr(self.album)
+        properties: dict[str, str] = await self.bridge.json_request(LinkPlayCommand.PLAYER_STATUS)
+
+        if not properties.get(PlayerAttribute.TITLE) and not properties.get(PlayerAttribute.ARTIST) and not properties.get(PlayerAttribute.ALBUM):
```

Comment:
Not sure how many other devices have this limitation, but when listening to radio presets I don't see any title, artist or album information from the `getPlayerStatusEx` endpoint.

The TCPUART response contains a title, which is why I'm falling back to this here.

Reply:
The `decode_hexstr` method is now called inside `decode_properties` a few lines further down

Reply:
And you're right, it's likely a firmware bug specific to my Edifier MS50A's.

Happy to remove the tcpuart fallback call from this PR and have that as a secondary discussion once TCPUART is implemented, if preferred?